nitro-protocol/test/contracts/NitroAdjudicator/concludeAndTransferAllAssets.test.ts

```diff
@@ -225,8 +226,10 @@ describe('concludeAndTransferAllAssets', () => {
       // Form transaction
       const tx = testNitroAdjudicator.concludeAndTransferAllAssets(
         getFixedPart(states[0]),
-        proof,
-        candidate,
+        proof as SignedVariablePart[],
+        candidate as SignedVariablePart,
```

Comment:
I've pushed a commit which removes the need for these type assertions, by making `separateProofAndCandidate` take a generic type.

Reply:
[a7aa42e](https://github.com/statechannels/go-nitro/pull/817/commits/a7aa42e958858d873c21ce978355c54ebfe30eef)

Reply:
Nice, great thanks!